auth.js

```diff
@@ -0,0 +1,137 @@
+const email = document.querySelector("#email");
+const password = document.querySelector("#password");
+const passwordConfirm = document.querySelector("#password_confirm");
+const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+
+const signUpButton = document.querySelector("#signUpButton");
+const loginButton = document.querySelector("#loginButton");
+signUpButton ? signUpButton.setAttribute("disabled", true) : loginButton.setAttribute("disabled", true);
+
+const passwordVisibleButtons = document.querySelectorAll("#passwordVisibleButton");
+
+const checkEmailData = (e) => {
+  const emailElementParent = e.target.parentNode;
+  const emailErrorMessegeEmpty = "이메일을 입력해 주세요.";
+  const emailErrorMessegeFormat = "잘못된 이메일 형식입니다.";
+
+  let emailErrorMsgElement = document.createElement("p");
```

Comment:
❗️ 수정요청
해당 변수에 재할당되고 있지 않으니 let 선언이 아닌 const 선언을 하시면 더 좋습니다.

```suggestion
  const emailErrorMsgElement = document.createElement("p");
```